README.md

```diff
@@ -1,369 +1,156 @@
-POA smart contracts allows maintainers of the bridges to 
-- Upgrade smart contracts by fixing potential security vulnerabilities in a smart contract with multisig contract
-- Add validators
-- Remove validators
-- Set required number of signatures from validators
-- Set daily limit of deposits on Home Network(left side) for native coin deposits(POA)
-- Set daily limit of withdrawals on Foreign Network(right side) for ERC20/ERC677 token transfers(POA20)
-- Set max limit per transaction for deposits on Home Network(left side) for native coin deposits(POA)
-- Set min amount per transaction for deposits on Home Network(left side) for native coin deposits(POA)
-- Set max limit per transaction for deposits on Foreign Network(right side) for ERC20/ERC677 token transfers(POA20)
-- Set min amount per transaction for deposits on Foreign Network(right side) for ERC20/ERC677 token transfers(POA20)
-- Storing Home Deposits in Home Bridge contract
-- Minting on Deposits of ERC20 compatible token on Foreign Network by validators
-- Burning on Withdrawals of ERC20 compatible token on Foreign Network by validators
-
-Bridge UI  features:
-- Show daily limits in both networks
-- Display all events in both networks
-- Filter events from specific block number on both sides
-- Find corresponding event on different sides of the network
+[![Gitter](https://badges.gitter.im/poanetwork/poa-bridge.svg)](https://gitter.im/poanetwork/poa-bridge?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+
+Welcome to the POA Bridge! Following is an overview of the POA Bridge and Bridge UI Application, as well as [basic instructions for getting started](#getting-started).
+
+
+## POA Bridge Overview
+
+The POA Bridge allows users to transfer assets between two chains in the Ethereum ecosystem. It is composed of several elements which are located in different POA Network repositories. 
+
+For a complete picture of the POA Bridge functionality, it is useful to explore each repository.
+
+**Bridge Elements**
+1. Bridge UI Application. A DApp interface to transfer tokens and coins between chains, located in this repository.
+2. [Bridge Smart Contracts](https://github.com/poanetwork/poa-bridge-contracts). Solidity contracts used to manage bridge validators, collect signatures, and confirm asset relay and disposal.
+3. [Bridge Oracle](https://github.com/poanetwork/token-bridge). An oracle written in NodeJS.
+4. [Bridge Monitor](https://github.com/poanetwork/bridge-monitor). A tool for checking balances and unprocessed events in bridged networks.
+5. [Bridge Deployment Playbooks](https://github.com/poanetwork/deployment-bridge). Manages configuration instructions for remote deployments and allows you to deploy separate bridge instances for validators.
+6. [Bridge Monitor](https://github.com/poanetwork/bridge-monitor).
+
+
+## Bridge UI Application
+
```

Comment:
I may add a picture of the redesigned bridge here

Reply:
Actually there is no much difference in UI between the previous and this version. Yes, please, add an example how the UI could look like.